lua/nvim-tree/actions/find-file.lua

```diff
@@ -13,6 +14,8 @@ function M.fn(fname)
   end
   running[fname] = true
 
+  local ps = log.profile_start("find file %s", fname)
```

Comment:
This is being invoked (with draw) twice when
```lua
		update_focused_file = {
			enable = true,
			update_cwd = true,
		},
```

That might explain some of the slowness in #791.

Either way, we probably should fix that ;)